src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-group/info-group.component.html

```diff
@@ -1,6 +1,26 @@
 <div class="row">
-  <span class="info-group-title">{{ groupTitle }}</span>
+  <div class="info-group-title"
+       [ngbPopover]="popInfoTemplate"
+       #popInfo="ngbPopover"
+       triggers="">
+       {{ groupTitle }}
+    <button type="button"
+            class="popover-icon"
```

Comment:
Somehow the following comment was not submitted on my last review...
This button has a background that is slightly different than the background color.
![image](https://user-images.githubusercontent.com/399326/92226977-15b5df80-ee95-11ea-95ec-2af54e043b3b.png)
Using `.btn` fixes this
```suggestion
            class="popover-icon btn"
```

I also wonder if we should add `.btn-link`
```suggestion
            class="popover-icon btn btn-link"
```
![image](https://user-images.githubusercontent.com/399326/92227174-6af1f100-ee95-11ea-8e52-936e76b524b6.png)
This way we have a icon similar to what we use in forms:
![image](https://user-images.githubusercontent.com/399326/92227206-79400d00-ee95-11ea-9dd2-8a055a81e688.png)

Reply:
You can also configure the padding using a class, that would remove the need for extra scss:
```suggestion
            class="popover-icon p-0"
```